includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -463,7 +464,7 @@ protected function is_exempt_from_validation( DOMNode $node ) {
 	 * @return bool Whether the node should have been removed, that is, that the node was sanitized for validity.
 	 */
 	public function remove_invalid_child( $node, $validation_error = [] ) {
-		if ( DevMode::isExemptFromValidation( $node ) ) {
+		if ( $this->is_exempt_from_validation( $node ) ) {
```

Comment:
I think this can be reverted based on my comment on the `sanitize` method below.